lasagna-spa.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>lasagna spagetii</title>
+    <link rel="stylesheet" href="lasagna-spa.css" />
+  </head>
+  <body>
+    <div class="container">
```

Comment:
Semantic here too

Reply:
https://www.codecademy.com/learn/learn-html/modules/learn-semantic-html/cheatsheet help you to know more and how to use semantic HTML tags